### ISSUE TITLE
Tests wanted!

### ISSUE THREAD

**Issue by adamsiwiec** [adamsiwiec/hexagon#2]
I'm not the best at writing tests, so I will leave it up to the community to decide what to do with testing,  but it needs to get done so somebody volunteer to write the tests.

**Comment by jesusgn90** [comment]
Is this project alive? I check the code tomorrow and see how to make it better if I can

**Comment by adamsiwiec** [comment]
Yes definitely, anything done to the code base would be appreciated, @jesusgn90, just submit a pull request!

**Comment by sukrosono** [comment]
hey guys, 2021
